### ISSUE TITLE
OSX build is not digitally signed

### ISSUE THREAD

**Issue by ingorichter** [zaggino/brackets-electron#63]
### Description

OSX complains about missing code signing of executable.
### Versions

1.8.1

[Please include the OS and what version of `Brackets-Electron` you're running.]
OSX 10.11.6
### Steps to Reproduce
1. Download the release build
2. Open DMG and copy to local Applications folder
3. Launch Brackets-Electron

**Expected behavior:** Launch the app without warning.

**Actual behavior:** Doesn't launch the app because of missing code signing signature.

**Workaround**
1. Open System Preferences
2. Open **Security & Privacy** and navigate to **General** tab
3. Change the setting to **Allow apps downloaded from:** to **Anywhere**

**Comment by zaggino** [comment]
Yeah, you need to be Apple Developer program member (or whatever it's called) to sign the app.
